NuRadioReco/detector/RNO_G/rnog_detector.py

```diff
@@ -1255,6 +1278,45 @@ def get_site_coordinates(self, station_id=None):
         return (72.57, -38.46)
 
 
+    def get_database(self):
+        """
+        Returns the database connection
+
+        Returns
+        -------
+
+        db: MongoClient
+            Returns the database connection
+        """
+        if self.__db is None:
+            self.logger.error("No database connection available. Return None")
+            return None
+
+        return self.__db
+
+
+    def get_component(self, collection="coax_cable", component="daq_drab_flower_2024_avg"):
```

Comment:
If this is the 'wrapper' around `get_component_data` you mentioned in #773 , with the intention that it should be used - please add a docstring; I'm happy for you to copy/update the documentation I added to explain this in #773 (maybe we merge that first?)